src/Octoshift/OctoLogger.cs

```diff
@@ -15,29 +16,55 @@ internal static class LogLevel
     public class OctoLogger
     {
         public bool Verbose { get; set; }
-        private readonly DateTime _logStartTime;
+        private readonly List<string> _secrets = new List<string>();
```

Comment:
I would use a `HashSet<string>` in order to not to add duplicate data.

Reply:
done